front/src/applications/stdcm/utils/formatStdcmConf.ts

```diff
@@ -1,22 +1,39 @@
-/* eslint-disable @typescript-eslint/no-unused-vars */
 import type { TFunction } from 'i18next';
 import { compact } from 'lodash';
 import type { Dispatch } from 'redux';
 
 import type {
+  PathItemLocation,
   PathfindingItem,
   PostTimetableByIdStdcmApiArg,
   TrainScheduleBase,
 } from 'common/api/osrdEditoastApi';
 import type { InfraState } from 'reducers/infra';
 import { setFailure } from 'reducers/main';
-import type { OsrdStdcmConfState, StandardAllowance } from 'reducers/osrdconf/types';
+import type { OsrdStdcmConfState, StandardAllowance, StdcmPathStep } from 'reducers/osrdconf/types';
 import { dateTimeFormatting } from 'utils/date';
 import { kmhToMs, mToMm, tToKg } from 'utils/physics';
 import { ISO8601Duration2sec, sec2ms } from 'utils/timeManipulation';
 
 import createMargin from './createMargin';
 
+const getStepLocation = (step: StdcmPathStep): PathItemLocation => {
+  if ('track' in step) {
+    return { track: step.track, offset: mToMm(step.offset) };
+  }
+  if ('operational_point' in step) {
+    return { operational_point: step.operational_point };
+  }
+  if ('trigram' in step) {
+    return { trigram: step.trigram, secondary_code: step.secondary_code };
```

Comment:
Hmm. I think we actually never use `PathStep.secondary_code` and instead always use `PathStep.ch`?

See: https://github.com/OpenRailAssociation/osrd/pull/8633

Reply:
Thanks for your PR :star_struck: